Author:         Trent W. Buck <trentbuck@gmail.com>
Forwarded:      not-needed
Bug-Debian:     http://lintian.debian.org/tags/extra-license-file.html
Description:    Prevent cabal installing COPYING in a quick and dirty way.
 Not forwarded because /usr/share/common-lincenses is Debian-specific.
Index: darcs-2.4.1/darcs.cabal
===================================================================
--- darcs-2.4.1.orig/darcs.cabal	2010-04-13 05:42:42.000000000 +1000
+++ darcs-2.4.1/darcs.cabal	2010-05-07 15:48:40.371169582 +1000
@@ -1,7 +1,6 @@
 Name:           darcs
 version:        2.4.1
 License:        GPL
-License-file:   COPYING
 Author:         David Roundy <droundy@darcs.net>, <darcs-users@darcs.net>
 Maintainer:     <darcs-users@darcs.net>
